contracts/zero-ex/contracts/src/vendor/ITakerCallback.sol

```diff
@@ -21,14 +21,20 @@ pragma solidity ^0.6.5;
 pragma experimental ABIEncoderV2;
 
 
-interface IERC1155OrderCallback {
+interface ITakerCallback {
 
-    /// @dev A taker callback function invoked in the ERC1155Feature between
-    ///      the maker -> taker transfer and the taker -> maker transfer.
+    /// @dev A taker callback function invoked in ERC721OrdersFeature and
+    ///      ERC1155OrdersFeature between the maker -> taker transfer and
+    ///      the taker -> maker transfer.
+    /// @param orderHash The hash of the order being filled when this
+    ///        callback is invoked.
     /// @param callbackData Arbitrary data used by this callback.
-    /// @return success The selector of this function (0x0ea5ba79),
+    /// @return success The selector of this function,
     ///         indicating that the callback succeeded.
-    function zeroExERC1155OrderCallback(bytes calldata callbackData)
+    function zeroExTakerCallback(
+        bytes32 orderHash,
```

Comment:
Consolidated the callbacks and threw in `orderHash` as a parameter as a security measure for integrators